src/fairchem/core/_cli_hydra.py

```diff
@@ -12,6 +12,7 @@
 from typing import TYPE_CHECKING
 
 import hydra
+from omegaconf import OmegaConf
```

Comment:
I dont think this is currently a dependency, should we add this to pyproject.toml?

Reply:
its installed through hydra